test-cases/README.md

```diff
@@ -0,0 +1,43 @@
+# Test Cases for Linters
+
+The files in this directory are for testing linting tools.
+
+## YAML Files
+
+* [test-yaml.yaml]():
+    * Missing the license
+    * Includes multiple syntax errors
+
+## Dockerfiles
+
+* [Dockerfile](): Example hello world build
```

Comment:
Did you mean these to be links, or something for a markdownlint test? :P

Reply:
I was going add the links but wanted to ask first if we found that useful.

Reply:
links have been added